ansible/ansible_collections/packaging/tools/plugins/modules/uv_tool.py

```
@@ -1,10 +1,140 @@
+Example playbook task::
+
+    - name: Install ruff with uv
+      packaging.tools.uv_tool:
+        name: ruff
+        version: 0.14.0
+        python: "3.12"
+"""
+
 from __future__ import annotations
 
+#!/usr/bin/python
```

Ruff (0.15.11)

[warning] Shebang should be at the beginning of the file

(EXE005)